ARGUS/ARGUS_UNet/utils/ARGUS_vfold_setup.py

```diff
@@ -0,0 +1,169 @@
+import os
+import numpy as np
+from glob import glob
+from monai.data import CacheDataset, DataLoader
+import ubelt as ub
+import pint
+Ureg = pint.UnitRegistry()
+
+def setup_ONSD_vfold_files(self, img_dir, anno_dir, p_prefix, n_prefix):
+    self.all_train_images = sorted(glob(os.path.join(img_dir, "*_cropM.mha")))
```

Comment:
The filename suffixes should be class member variables, defined in the experiment's config file.

Reply:
Done!